includes/class-sensei-autoloader.php

```diff
@@ -205,6 +205,8 @@ public function initialize_class_file_map() {
 			'Sensei\Blocks\Course_Theme\Prev_Next_Lesson' => 'blocks/course-theme/class-prev-next-lesson.php',
 			'Sensei\Blocks\Course_Theme\Prev_Lesson'      => 'blocks/course-theme/class-prev-lesson.php',
 			'Sensei\Blocks\Course_Theme\Next_Lesson'      => 'blocks/course-theme/class-next-lesson.php',
+			'Sensei\Blocks\Course_Theme\Course_Title'     => 'blocks/course-theme/class-course-title.php',
+			'Sensei\Blocks\Course_Theme\Site_Logo'        => 'blocks/course-theme/class-site-logo.php',
```

Comment:
Just a question for @sruj09 about the purpose of that. Is that logo in the top bar a site logo or a course logo?

Reply:
> Just a question for @sruj09 about the purpose of that. Is that logo in the top bar a site logo or a course logo?

Site logo. I done imagine that people will have a logo by course. We were taking on this Slack thread about using the same as the [site logo](https://a8c.slack.com/archives/C02KZBEJ30D/p1637697936034800?thread_ts=1637689228.026600&cid=C02KZBEJ30D) for this.